tests/datasource/data_connector/test_sql_data_connector.py

```diff
@@ -1103,6 +1104,92 @@ def test_more_complex_instantiation_of_ConfiguredAssetSqlDataConnector(
     }
 
 
+def test_more_complex_instantiation_of_ConfiguredAssetSqlDataConnector_include_schema_name(
+    test_cases_for_sql_data_connector_sqlite_execution_engine,
+):
+    my_data_connector = ConfiguredAssetSqlDataConnector(
```

Comment:
I'm not convinced a type hint is required here, but we have been doing it elsewhere.